global_vars.tf

```diff
@@ -60,6 +61,11 @@ variable "private_route_table_ids" {
   description = "Route tables with route to NAT gateway"
 }
 
+variable "ons_access_ips" {
+  type        = "list"
+  description = "List of IP's or IP ranges to allow access from ONS"
```

Comment:
We should default this to `[]` so that we don't need to provide it if it's not used